src/main/resources/InstructorSampleData.json

```diff
@@ -276,10 +276,10 @@
         "value": "Please give your feedback based on the following questions."
       },
       "createdTime": "2012-04-01 11:59 PM UTC",
-      "startTime": "2012-04-01 11:59 PM UTC",
-      "endTime": "2037-04-02 11:59 PM UTC",
-      "sessionVisibleFromTime": "2012-04-01 11:59 PM UTC",
-      "resultsVisibleFromTime": "1970-01-01 00:00 AM UTC",
+      "startTime": "2012-04-01 3:59 PM UTC",
+      "endTime": "2037-04-02 3:59 PM UTC",
+      "sessionVisibleFromTime": "2012-04-01 3:59 PM UTC",
+      "resultsVisibleFromTime": "1970-01-01 12:00 AM UTC",
```

Comment:
Not a big deal but this should not be touched?

Reply:
I'm not sure `1970-01-01 00:00 AM UTC` is even a valid date (since hours can only be 1-12 when using AM/PM)? I got a parse error in my python script and had to manually fix it. I think we've been relying on JavaScript's default date parser being more forgiving.

Reply:
I'm surprised this works; it shouldn't:
https://github.com/TEAMMATES/teammates/blob/20cf5a8c08456b5600d1c64dc15f29b98b9aa3f4/src/main/java/teammates/common/util/Const.java#L104-L111

`TimeHelper#convertToDate` is using `SimpleDateFormat` with `yyyy-MM-dd h:mm a Z` as the date format. `h` is clearly defined as `Hour in am/pm (1-12)` in Java documentation: https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

Reply:
All the other JSON data files have it in the correct format ([link](https://github.com/TEAMMATES/teammates/search?utf8=%E2%9C%93&q=%2212%3A00+AM%22&type=)); it's only this one instance which I believe was manually inputted instead of generated that's wrong.